imagebuildah/chroot_symlink.go

```diff
@@ -1,16 +1,13 @@
 package imagebuildah
```

Comment:
Can't you just add a #linux to this, and then implement the windows global interfaces?